ru.proninyaroslav.libretrack.yaml

```diff
@@ -0,0 +1,99 @@
+app-id: ru.proninyaroslav.libretrack
+runtime: org.gnome.Platform
+runtime-version: '40'
+sdk: org.gnome.Sdk
+command: libretrack
+separate-locales: false
+finish-args:
+  # Required due to being a GUI application
+  - --socket=x11
+  - --socket=fallback-x11
+  # Required to make sure x11 performance is achived on all platforms
+  # At least that's what the legends tell. it might be worth experimenting
+  # with dropping this permission.
+  - --share=ipc
+  # Required for experimental wayland support
+  - --socket=wayland
+  # Required to provide notification functionality
+  - --socket=pulseaudio
+  # As a tracking application, networking is required
+  - --share=network
+  - --device=all
+  # Required for notifications in various desktop environments
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  # Required due to KDE's special way of handling notifications
+  - --own-name=org.kde.StatusNotifierItem-2-1
+  # Required for advanced input methods e.g. writing CJK languages
+  - --talk-name=org.freedesktop.portal.Fcitx
+  # Required for correct work of Flutter packages, which use system DBus
+  # for example https://pub.dev/packages/connectivity_plus
+  - --socket=system-bus
+  # TODO: Temporary fix https://github.com/canonical/dbus.dart/issues/257
+  - --socket=session-bus
+  # Required for storing Floor database https://pub.dev/packages/floor
+  - --persist=.dart_tool
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /lib/*.a
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - shared-modules/libsecret/libsecret.json
+  - name: libjsoncpp
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=release
+      - -DBUILD_SHARED_LIBS=ON
+    sources:
+      - type: archive
+        url: https://github.com/open-source-parsers/jsoncpp/archive/refs/tags/1.7.5.tar.gz
+        sha256: 4338c6cab8af8dee6cdfd54e6218bd0533785f552c6162bb083f8dd28bf8fbbe
+  - name: libretrack
+    buildsystem: simple
+    only-arches:
+      - x86_64
+      - aarch64
+    build-commands:
+      - mkdir -p /app
+      - cp -r libretrack /app/libretrack
+      - chmod +x /app/libretrack/libretrack
+      - install -Dm644 /app/libretrack/data/icons/app-icon.svg /app/share/icons/hicolor/scalable/apps/ru.proninyaroslav.libretrack.svg
+      - mkdir -p /app/bin /app/lib
+      - ln -s /usr/lib/x86_64-linux-gnu/libsqlite3.so.0 /app/lib/libsqlite3.so
```

Comment:
this will fail on aarch64.
Also why do you need it?

Reply:
`libsqlite3` is not automatically linked to the `/lib` dir. At runtime, I will get an exception because the app can't find the library. I will add a check.